src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -32,6 +32,50 @@ public static void Ctor_CharSpan(char[] valueArray, int startIndex, int length,
             Assert.Equal(expected, new string(span));
         }
 
+        [Fact]
+        public static void Create_InvalidArguments_Throw()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("action", () => string.Create(-1, 0, null));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("length", () => string.Create(-1, 0, (span, state) => { }));
```

Comment:
Not that I expect anything weird to happen, but seeing the empty action here made me wonder if it's worth having a non-zero length with the empty action. (which should just be equal to `new string((char)0, length)`

Actually, I guess really it's ensuring that we're using cleared memory, since the other tests set every char position we don't have that assurance.

Reply:
Good suggestion, @bartonjs.  I added a test for it.